scripts/obtain/uninstall/dotnet-uninstall-rpm-packages.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
```

Comment:
I think a separate uninstall script for packages is a bad idea. I left a more detailed comment here: https://github.com/dotnet/cli/issues/6345#issuecomment-295918725